ccd-definition/CaseField.json

```diff
@@ -12,7 +12,7 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "ID": "caseName",
     "Label": "Case name",
-    "HintText": "Include the Local Authority name and respondent’s last name. For example, Endley Council v Smith/Tate/Jones",
+    "HintText": "Include the Local Authority name and respondent's last name. For example, Endley Council v Smith/Tate/Jones",
```

Comment:
Think this might be slightly out of scope of the bug ticket but I guess it doesn't hurt to be consistent